### DISCUSSION TITLE
what is the recommended set up in k8s?

### DISCUSSION THREAD

**Discussion by sivankumar86** [airbytehq/airbyte#36585] [General]
Is it recommended to setup "CONTAINER_ORCHESTRATOR_ENABLED" = true  in k8s ? I have checked helm chart and there is no option to set up "CONTAINER_ORCHESTRATOR_ENABLED" param.

**Comment by azorej** [comment]
As far as I know it's enabled by default for oss helm installation
And I'm not sure it's safe to disable it (I think Airbyte team deprecated/will deprecate CONTAINER_ORCHESTRATOR_ENABLED=false mode)

**Comment by sivankumar86** [comment]
It would be great if there is an  architect document which talk about k8s airbyte internals as it is differ from VM installation

[a comment by an automated account is not shown]
